workflow/Snakefile

```diff
@@ -20,6 +20,7 @@ include: "rules/ref.smk"
 include: "rules/quant.smk"
 include: "rules/diffexp.smk"
 include: "rules/enrichment.smk"
+include: "rules/plot.smk"
```

Comment:
Do we really need another smk-file for this? Shouldn't this fit into `diffexp.smk`? Also, if we have a `plot.smk`, I would expect all rules for plots to be found there, so this could even be confusing.

Reply:
Well, it's not expensive to have one more smk file ;)
I thought of moving all plot related rules to plot.smk, to have them all in one place. However, some plots are part/byproduct of e.g. sleuth routines, so perhaps that's not feasible.
Also, we might want to add more (interactive/custom) plots in the future. To go even more overboard with this, we might have a whole "plot" subdirectory of smk files for different parts of the workflow.

Anyways, for now I'll either move all plotting rules into plot.smk or move that single rule to diffexp.smk, probably the latter. Thoughts?

Reply:
For now, diffexp.smk seems to be the right place indeed.

Reply:
Can you move it there? Then, this is ready for merging I think.